app/src/lib/stats/stats-store.ts

```diff
@@ -514,7 +515,7 @@ export class StatsStore implements IStatsStore {
 
     // isInApplicationsFolder is undefined when not running on Darwin
     const launchedFromApplicationsFolder =
-      remote.app.isInApplicationsFolder?.() ?? null
+      (await isInApplicationFolder()) ?? null
```

Comment:
It'd be great if we could avoid the IPC roundtrip and event loop dispatch here when we're not on macOS.
```suggestion
      ? (await isInApplicationFolder()) ?? null
      : null
```

Reply:
Makes sense. I also made the ipc call return null instead of undefined so we don't need the ?? anymore.